h2o-core/src/test/java/water/util/JavaVersionUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package water.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import water.TestUtil;
+
+import static water.util.JavaVersionUtils.JAVA_VERSION;
+
+/**
+ * Java version string parser test.
+ */
+public class JavaVersionUtilsTest extends TestUtil {
+
+    @BeforeClass
+    static public void setup() {  stall_till_cloudsize(1); }
```

Comment:
No need - it could be just a regular junit test, it doesn't have inherit from TestUtil